docs/en_US/Compression/QuickStart.rst

```diff
@@ -45,7 +45,7 @@ After training, you get accuracy of the pruned model. You can export model weigh
 
    pruner.export_model(model_path='pruned_vgg19_cifar10.pth', mask_path='mask_vgg19_cifar10.pth')
 
-The complete code of model compression examples can be found :githublink:`here <examples/model_compress/pruning/model_prune_torch.py>`.
+Please refer :githublink:`mnist example <examples/model_compress/pruning/mnist_torch.py>` for quick start.
```

Comment:
there is no such file

Reply:
Thanks for pointing it out. The link has been updated.